lib/elasticsearch/elastic-query-builder.js

```diff
@@ -505,38 +556,7 @@ class ElasticQueryBuilder {
     const simpleMatchFilters = Object.keys(this.request.params.filters)
       .filter((k) => FILTER_CONFIG[k].operator === 'match')
 
-    filterClausesWithPaths = filterClausesWithPaths.concat(simpleMatchFilters.map((prop) => {
-      const config = FILTER_CONFIG[prop]
-
-      let value = this.request.params.filters[prop]
-
-      // This builds a filter cause from the value:
-      const buildClause = (value) => {
-        // If filtering on a packed field and value isn't a packed value:
-        if (config.operator === 'match' && value.indexOf('||') < 0 && config.field.match(/_packed$/)) {
```

Comment:
I removed these references to `config.operator === 'match'` because we are passing in an array that is already filtered on that condition.